
public namespace typeinf5;
import common -;

function int foo(int x)
{
  varray{int} v;
  v.resize(5, int());
  mutable& y = v[3];
  y += 1;
  y += x;
  v.resize(10, int()); // error
  return v[3];
}

exception::set_stack_trace_limit(0);
try {
  println(foo(3));
} catch (c_exception ex) {
  println("caught", ex);
}

